person.rb

```diff
@@ -0,0 +1,47 @@
+class Person
+  def initialize (age, name = 'Unknown', parent_permission = true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
```

Comment:
- [ ] I will implore you to add the GETTERS and SETTERS which is need name and age you can use the shortcut of attri_accessor `attr_accessor :name, :age` and use `attr_reader :id` since we are only GETTING and not SETTING.